source/body.ts

```diff
@@ -24,6 +24,7 @@ export type MediaType = typeof MEDIA_TYPES[number];
 
 export type TextBody = {
 	readonly text: string;
+	readonly entities?: MessageEntity[];
```

Comment:
```suggestion
	readonly entities?: ReadonlyArray<ReadonlyDeep<MessageEntity>>;
```

Using readonly things would resolve the `@typescript-eslint/prefer-readonly-parameter-types` lint. I'm not entirely sure whether that's a good thing in this case. But it probably is as entities are not modified after this point so they can be easily set, and it should just work. For the MediaBody this lint is ignored, so I'm not sure whether that should be changed or if this should be ignored in this case too.

(also applies for the `entities` in `MediaBody`, but I omitted it in the review as its basically the same question again)

Reply:
Thanks for the prompt reply.

I also tried that, but then I get a lot of errors in `send-menu.ts`.

Here's a snippet of the errors I get:
![image](https://github.com/user-attachments/assets/7f7124d0-d650-4d43-8384-6ae6d8105c7c)

Reply:
Guess we can just ignore the readonly thingy then, similar to the `MediaBody`… a bit sad but the `MessageEntity` isn't a type specified in grammy-inline-menu directly so its kinda out of scope already.

Reply:
OK, thanks a lot, I disabled this inspection for this and fixed the tests.
The unit tests are now successful!